src/Login/Login.jsx

```diff
@@ -33,43 +34,50 @@ const Login = ({ history }) => {
   }
 
   return (
-    <div className="col-md-auto">
-      <h1>Tindoori</h1>
-      <Form onSubmit={handleLogin}>
-        <Form.Group>
-          <Form.Label>
-            Email
-            <Form.Control
-              type="email"
-              id="email"
-              placeholder="Enter email"
-              name="email"
-            />
-          </Form.Label>
-        </Form.Group>
-        <Form.Group>
-          <Form.Label>
-            Password
-            <Form.Control
-              type="password"
-              id="password"
-              placeholder="Password"
-              name="password"
-            />
-          </Form.Label>
-        </Form.Group>
-        {error && (
-          <Alert variant="danger" role="alert">
-            {error.message}
-          </Alert>
-        )}
+    <Card id="registration-card">
+      <div id="feed">
```

Comment:
Why is there a feed ID?

Reply:
I also wonder

Reply:
![image](https://user-images.githubusercontent.com/33896659/115867771-c92b0e80-a43b-11eb-8570-57394e1b4e0c.png)
This happens when you remove feed